addon/webextension/selector/ui.js

```diff
@@ -95,16 +95,28 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
     return window.downloadOnly;
   }
 
-  function renderDownloadNotice() {
-    let notice = makeEl("div", "download-only");
+  // the download notice is rendered in iframes that match the document height
+  // or the window height. If parent iframe matches widnow height, pass in true
+  function renderDownloadNotice(initAtBottom = false) {
+    let notice = makeEl("table", "download-only");
     notice.innerHTML = `
```

Comment:
idk, kinda weird to stuff divs directly inside a table element, but I guess if it works...